src/script.js

```diff
@@ -0,0 +1,122 @@
+import { format } from 'date-fns';
+import { form, alertContainer } from './dom';
+
+const cityKey = 'QqFCfLuVXyO7oxFlgu4nWgRjh5hYjgVA';
+const cityBase = 'http://dataservice.accuweather.com/locations/v1/cities/search';
+const weatherKey = '67c0476c31125a1dbcfdd521a5a88903';
+
+// get city data from accuweather api
+
+function weatherCheck() {
+  return {
+
+    city: '',
+
+    async getCity(city, base, key) {
+      const query = `?apikey=${key}&q=${city}`;
+
+      const response = await fetch(base + query);
+      const data = await response.json();
+
+      return data;
+    },
+    async getWeather(city, key) {
+      const WeatherBase = `http://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${key}`;
+      const response = await fetch(WeatherBase);
+      const data = await response.json();
+
+      return data;
+    },
+    // setup templates for render function
+    templates() {
+      return {
+        searchUl(data) {
+          return ` <li><span>${data.EnglishName}</span><span>${data.Country.EnglishName}</span></li>`;
+        },
+        detailsUl(data) {
+          return `<li><span>${data.name}</span><span>${data.sys.country}</span></li>
+                            <li><span>${data.weather[0].main}</span><span>${data.weather[0].description}</span></li>
+                            <li><span>Humidity</span><span>${data.main.humidity}</span></li>
+                            <li><span>Temperature</span><span> ${Math.ceil(data.main.temp - 273.15)}&deg;</span></li>`;
+        },
+        mainWeather(data) {
+          const dating = format(new Date(), 'MM/dd/yyyy');
+          return `
+                                <h2 class="deg font-weight-bolder display-1">${Math.ceil(data.main.temp - 273.15)}&deg;</h2>
+                                
+                                <div class="h6 location-time mx-3">
+                                    <h5 class="location-text display-4">${data.name} / ${data.sys.country}</h5>
+                                   ${dating}
+                                   
+                                </div>
+                                <div class="location-icons">
+                                <i class="icofont-clouds"></i>
+                                    <p class="text-capitalize">${data.weather[0].description}</p>
+                                </div>
+                            `;
+        },
+
+      };
+    },
+    // function to render things on the screen
+    render(node, clas, template) {
+      const container = document.createElement('ul');
+
+      container.classList = clas;
+      node.innerHTML = '';
+      container.innerHTML = template;
+      node.appendChild(container);
```

Comment:
- [ ] Make sure that DOM manipulation such as this and the document queries below are in another file inside their own functions, then you can call the functions here.